website/static/js/_anonymousLogTexts.js

```diff
@@ -0,0 +1,132 @@
+var anonymousLogActions = {
```

Comment:
Minor: it would be more consistent to name this file `anonymousLogActionList.js`, but this doesn't matter much.